cupy/cudnn.pyx

```diff
@@ -581,6 +581,14 @@ def rnn_backward_weights_ex(
     return dw
 
 
+def create_activation_descriptor(mode, nan_prop_mode=cudnn.CUDNN_PROPAGATE_NAN,
```

Comment:
Is this related to NHWC support?

Reply:
Yes, it will be used by fast cuDNN's batch normalization implementation for NHWC layout. Please see [here ](https://github.com/anaruse/chainer/blob/e147e887cc3ec6170c0ddc1d60a100080f1a7fa7/chainer/functions/normalization/batch_normalization.py#L244).

Reply:
In addition, this create_activation_descriptor will be also necessary when cuDNN's fused operations are used.